src/lib/offlineMutationQueue.ts

```
@@ -206,23 +206,24 @@ export async function replayOfflineMutations(): Promise<{
-          isSuccess = !error;
-        } else {
-          const { error } = await supabase
-            .from("post_reactions")
-            .delete()
-            .eq("post_id", item.payload.postId)
-            .eq("user_id", userId)
-            .eq("emoji", emoji);
-          isSuccess = !error;
-        }
-      } else if (item.type === "rsvp" && item.payload?.eventId) {
+if (item.action === "add") {
```

ESLint

[error] Insert `········`

(prettier/prettier)